### PR TITLE
Some fixes from FreifunkMYK

### PR DESCRIPTION
**Pull request by awlx**
Maybe we should have a look, as sadly there isn't a PR yet we maybe want to check for ourselves :).

### CODE REVIEW

File: wgkex/worker/mqtt.py [hunk 1]
```diff
@@ -53,7 +61,7 @@ def on_message(client: mqtt.Client, userdata: Any, message: mqtt.MQTTMessage) ->
         message: The MQTT message.
     """
     # TODO(ruairi): Check bounds and raise exception here.
-    domain = re.search(r"/.*ffmuc_(\w+)/", message.topic).group(1)
+    domain = re.search(r"^wireguard\/ff(\w+)\/", message.topic).group(1)
```
Comment (awlx) [inline wgkex/worker/mqtt.py:64]: We need to make this a config option, otherwise this will be a mess.

File: wgkex/worker/netlink.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def wg_flush_stale_peers(domain: str) -> List[Dict]:
         The peers which we can remove.
     """
     stale_clients = [
-        stale_client for stale_client in find_stale_wireguard_clients("wg-" + domain)
+        stale_client for stale_client in find_stale_wireguard_clients("wg" + domain)
```
Comment (awlx) [inline wgkex/worker/netlink.py:70]: we need this hyphen ... maybe also something to add to configuration like "wg_interface_prefix"?

File: wgkex/worker/netlink.py [hunk 3]
```diff
@@ -49,12 +49,12 @@ def lladdr(self) -> str:
     @property
     def vx_interface(self) -> str:
         """Returns the name of the VxLAN interface associated with this lladdr."""
-        return f"vx-{self.domain}"
+        return f"vx{self.domain}"
```
Comment (rucarrol-goog) [inline wgkex/worker/netlink.py:52]: If we are changing the format here - we should be a bit careful, as we re-use this elsewhere. Could we consider putting these prefixes into the config file?

**Review by rucarrol-goog — CHANGES_REQUESTED** [review]

**Comment by krombel** [comment]
This PR is based on the main branch of a fork. So this is just for reference. Features can be taken from here but this is probably never going to land directly in our main branch. Marking as draft therefore

**Pull request closed without merging**